src/integrations/unixIntegrationManager.ts

```diff
@@ -2,18 +2,18 @@ import fs from 'fs';
 import os from 'os';
 import path from 'path';
 
-// Manages integrations, which include standalone binaries such as
-// kubectl and helm, as well as docker CLI plugins such as docker-compose
-// and docker-buildx. Integrations take the form of symlinks from
-// the Rancher Desktop installation to two separate directories:
-// the "integrations directory", which should be in the user's path somehow,
-// and the "docker CLI plugins directory", which is the directory that
+import { IntegrationManager } from '@/integrations/integrationManager';
+
+// Manages integrations for Unix-like operating systems. Integrations take
```

Comment:
Just realized — this isn't a proper JSDoc-style comment (needs `/**` or, I think, `///`). Same with the comment for `IntegrationManger`.

(This just means my IDE doesn't show the class comment on mouseover; it's not really critical, just a nice-to-have.)